impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -69,6 +71,49 @@ public void kafkaClusterZookeeperMode() throws Exception {
         }
     }
 
+    @Test
+    public void kafkaClusterZookeeperAddBroker() throws Exception {
+        int brokersNum = 1;
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder()
+                .testInfo(testInfo)
+                .brokersNum(brokersNum)
+                .kraftMode(true)
+                .build())) {
+            cluster.start();
+            assertThat(cluster.getNumOfBrokers()).isEqualTo(brokersNum);
+            assertThat(cluster.getBootstrapServers().split(",")).hasSize(brokersNum);
+
+            verifyRecordRoundTrip(brokersNum, cluster);
+
+            int nodeId = cluster.addBroker();
+            assertThat(nodeId).isEqualTo(1);
+            assertThat(cluster.getNumOfBrokers()).isEqualTo(brokersNum + 1);
+            assertThat(cluster.getBootstrapServers().split(",")).hasSize(brokersNum + 1);
+            verifyRecordRoundTrip(brokersNum + 1, cluster);
+        }
+    }
+
+    @Test
+    public void kafkaClusterZookeeperRemoveBroker() throws Exception {
```

Comment:
Shouldn't be necessary to create another test to remove a broker in kraftMode or do you consider that it is fine testing only for zookeeper?

Reply:
I added a new test to cover the case.